src/js/chat.js

```diff
@@ -584,7 +582,10 @@ function initSock() {
                     break;
                 case "NOTICE":
                 case "PRIVMSG":
-                    postMessage(cmd.params[1], false);
+                    if (cmd.params[0].indexOf('#') == -1 && cmd.params[0].indexOf('&') == -1)
```

Comment:
Not sure why you added in the extra if/else here - looks like leftover code from your WIP PM system? I think you might want to remove this.

Reply:
Was sure I removed it... will do both now